src/cf-typography/usage.md

```diff
@@ -49,6 +50,22 @@ project by duplicating the variable `@key: value`.
 @webfont-demi: 'AvenirNextLTW01-Demi';
 ```
 
+### Font source variables
+
+Use this variable to specify where the fonts files used in the `@webfont-*`
+variables are located. Can be either a relative or absolute path.
+
+```
+@cf-fonts-path: '/fonts'
+```
+
+If you want to load fonts from fonts.net, set the following variable to `true`,
+otherwise set it to `false` to use the self-hosted font path:
+
+```
+@use-font-net: true;
```

Comment:
How about we rename this variable to `@use-font-cdn` to make it agnostic, in case we change providers in the future?